docs/products/mysql/howto/connect-using-mysqlx-with-python.rst

```diff
@@ -0,0 +1,61 @@
+Connect to MySQL using MySQLx with Python
+=========================================
+
+Enabling MySQLx protocol support allows customers to use their MySQL instance as a document store. This example shows how to connect to your Aiven for MySQL instance using MySQLx protocol.
+
+.. note::
+
+    MySQL initially provided support for X-DevAPI (MySQLx) in v5.7.12 as an optional extension that can be installed by the user. On the MySQL v8.0+, the X-DevAPI is supported by default.
+
+Variables
+'''''''''
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``SERVICE_URI``
+     - Connection information about your Aiven for MySQLx service.
+   * - ``MYSQLX_PASSWORD``
+     - Password for ``avnadmin`` user.
+   * - ``MYSQLX_USER``
+   * - Your Aiven **User** information.
+
+Prerequisites
+'''''''''''''
+
+* Python 3.7 or later
+  
+* A ``mysqlx`` python library installed::
+     
+     pip install mysql-connector-python
+
+* An Aiven account with a service running. Click on **MySQLx** tab under **Connection information**, and copy ``SERVICE_URI`` and ``Port`` information.
```

Comment:
if SERVICE_URI already includes port then copying port separately not needed?

Reply:
true, removed